scipy/integrate/_quadpack_py.py

```diff
@@ -583,15 +585,30 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
 
     Examples
     --------
-
     Compute the double integral of ``x * y**2`` over the box
     ``x`` ranging from 0 to 2 and ``y`` ranging from 0 to 1.
+    That is, :math:`\\int^{x=2}_{x=0} \\int^{y=1}_{y=0} x y^2 dy dx`.
 
     >>> from scipy import integrate
     >>> f = lambda y, x: x*y**2
-    >>> integrate.dblquad(f, 0, 2, lambda x: 0, lambda x: 1)
+    >>> integrate.dblquad(f, 0, 2, 0, 1)
         (0.6666666666666667, 7.401486830834377e-15)
 
+    Calculate :math:`\\int^{x=\\pi/4}_{x=0} \\int^{y=\\cos(x)}_{y=\\sin(x)} 1 dy dx`.
+
+    >>> from numpy import pi, cos, sin
```

Comment:
We don't need to import NumPy in example. `np` is available without import, please use it below.